backend/tests/controller/GenerateReportController.test.ts

```diff
@@ -16,7 +16,7 @@ describe("GenerateReporter", () => {
       .post("/generateReporter")
       .send(new GenerateReportRequest());
     expect(response.status).equal(200);
-    expect(response.body).to.deep.equal(new GenerateReporterResponse());
+    expect(response.body).to.deep.equal({});
```

Comment:
why we got an empty object on response

Reply:
restored that. the newly generated instance of GenerateReporterResponse's default values for all fields are undefined, so its actually the same to an empty object.